internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```
@@ -0,0 +1,50 @@
+ * `/SPLUNK_INGEST_URL:` - URL of the Splunk ingest  (e.g. `https://ingest.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_INGEST_URL` (if it exists) will be used. Otherwise, defaults to `https://ingest.us0.signalfx.com`.
+ * `/PLUNK_API_URL` - URL of the API endpoint (e.g. `https://api.YOUR_SPLUNK_REALM.signalfx.com`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` file. If the parameter is not specified, the value found in `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_API_URL` (if it exists) will be used. Otherwise, defaults to `https://api.us0.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\Splunk\OpenTelemetry Collector`.
+ * `/SPLUNK_HEC_TOKEN` - Splunk HEC is HTTP Event Collecter token which will collect the metrics and logs of host system in to splunk.. If the parameter is specified, the token will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\hec-token` file.
+ * `/SPLUNK_HEC_URL` - URL of Splunk HEC (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v1/log`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_HEC_URL` file.
+ * `/SPLUNK_TRACE_URL` - Trace url is end point where apllication traces will be collected. URL of Splunk TRACE (e.g. `https://ingest.$SPLUNK_REALM.signalfx.com/v2/trace`). The URL will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_TRACE_URL` file.
+ * `/SPLUNK_REALM` - The parameter will be saved to the `\ProgramData\Splunk\OpenTelemetry Collector\SPLUNK_REALM` file. If not specified default is ("us0").
+ * `/SPLUNK_BUNDLE_DIR` - The path to the Agent bundle directory. The default path is provided by the collector package. If the specified path is changed from the default value, the path should be an existing directory on the node. The SPLUNK_BUNDLE_DIR environment variable will be set to this value for the collector service.
+ 
+
+To pass parameters, use `--params "''"` (e.g. `choco install signalfx-agent [choco options] --params="'/access_token:YOUR_SPLUNK_ACCESS_TOKEN /SPLUNK_INGEST_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com /SPLUNK_API_URL:https://api.YOUR_SPLUNK_REALM.signalfx.com /install_dir:YOUR_INSTALL_DIR /SPLUNK_HEC_TOKEN:YOUR_SPLUNK_HEC_TOKEN /SPLUNK_HEC_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v1/log /SPLUNK_TRACE_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v2/trace /SPLUNK_REALM:YOUR_SPLUNK_REALM /SPLUNK_BUNDLE_DIR:YOUR_DIR_FOR_AGENT_BUNDLE'"`).

```

I believe this correction is needed, but overall it'd be preferable to just enter the token and realm to make this simpler.

```suggestion
To pass parameters, use `--params "''"` (e.g. `choco install signalfx-agent [choco options] --params="'/SPLUNK_ACCESS_TOKEN:YOUR_SPLUNK_ACCESS_TOKEN /SPLUNK_INGEST_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com /SPLUNK_API_URL:https://api.YOUR_SPLUNK_REALM.signalfx.com /install_dir:YOUR_INSTALL_DIR /SPLUNK_HEC_TOKEN:YOUR_SPLUNK_HEC_TOKEN /SPLUNK_HEC_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v1/log /SPLUNK_TRACE_URL:https://ingest.YOUR_SPLUNK_REALM.signalfx.com/v2/trace /SPLUNK_REALM:YOUR_SPLUNK_REALM /SPLUNK_BUNDLE_DIR:YOUR_DIR_FOR_AGENT_BUNDLE'"`).

```